src/pools/Pool.cairo

```diff
@@ -632,7 +637,7 @@ func getPoolConfig{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_p
 func getNextPrice{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}() -> (
     next_price: Uint256
```

Comment:
Shouldn't this function have the sign as a parameter here? If a client wants to query the trade pool they should be able to get both directions.